crates/bevy_render/src/camera/camera.rs

```diff
@@ -130,18 +133,37 @@ impl Camera {
         world_position: Vec3,
     ) -> Option<Vec2> {
         let window_size = self.target.get_logical_size(windows, images)?;
+
+        if let Some(ndc_space_coords) = self.world_to_ndc(camera_transform, world_position) {
```

Comment:
This would benefit from `?` syntax.

Reply:
Specifically: `let ndc_space_coords = self.world_to_ndc(camera_transform, world_position)?;`